bindata/configmaps/console-configmap.yaml

```diff
@@ -4,4 +4,5 @@ metadata:
   namespace: openshift-console
   labels:
     app: "console"
-  annotations: {}
+  annotations:
+    capability.openshift.io/name: console
```

Comment:
I don't believe this needs to be on anything the operator itself creates (in other words anything in `bindata`). Just the YAML files in `manifests`.